### PR TITLE
css updated to reduce heading margin

### PR DESCRIPTION
**Pull request by Muhammad-Ali-Ahmad**
- [ yes ] Unit tests passing
- [ yes ] Coverage reached
- [ yes ] Acceptance tests passing
- [ yes ] Reviewed

### CODE REVIEW

File: app/assets/sass/tai19032019.scss [hunk 1]
```diff
@@ -167,4 +167,8 @@ hr {
   bottom: 0;
   right: 0;
   z-index: 9999;
-}
\ No newline at end of file
+}
+
+.incomeSourceSummary .heading-48 {
+    margin-bottom: .625em;
```
Comment (timsb) [inline app/assets/sass/tai19032019.scss]: What font-size does the resolve to? Wouldn't it be better to change the class used to another heading size?

File: app/views/IncomeSourceSummary.scala.html [hunk 2]
```diff
@@ -76,7 +76,7 @@
     title = title,
     authedUser = Some(user),
     pageHeader = Some(header),
-    articleClasses = None,
+    articleClasses = Some("incomeSourceSummary"),
```
Comment (timsb) [inline app/views/IncomeSourceSummary.scala.html]: Do you still need this class?

Comment (Muhammad-Ali-Ahmad) [reply app/views/IncomeSourceSummary.scala.html]: I won't if Rachel approves the design that only requires class update

File: app/assets/sass/tai19032019.scss (hunk 1 above)
Comment (Muhammad-Ali-Ahmad) [reply app/assets/sass/tai19032019.scss]: Updated as per suggestion

File: app/views/IncomeSourceSummary.scala.html (hunk 2 above)
Comment (Muhammad-Ali-Ahmad) [reply app/views/IncomeSourceSummary.scala.html]: updated as per suggestion

**Pull request merged**